liberty-managed/src/test/java/io/openliberty/arquillian/managed/needsmanagementmbeans/BazServlet.java

```diff
@@ -1,3 +1,18 @@
+/*
+ * Copyright 2018, IBM Corporation
```

Comment:
this is missing the ", and individual contributors" if it continues below with the "by the @authors tag"